Riot/Categories/UITableViewCell.swift

```diff
@@ -42,4 +42,9 @@ extension UITableViewCell {
     @objc func vc_setAccessoryDisclosureIndicatorWithCurrentTheme() {
         self.vc_setAccessoryDisclosureIndicator(withTheme: ThemeService.shared().theme)
     }
+    
+    /// Hide cell separators
+    @objc func vc_hideSeparator() {
```

Comment:
It's used at https://github.com/vector-im/element-ios/blob/e5f511117befb2d08e438a436ded8788df041196/Riot/Modules/Room/RoomInfo/RoomInfoList/RoomInfoListViewController.swift#L292. We cannot use `tableView.separatorStyle = .none` because we want to hide separator only for specific cells.

Reply:
Not used anymore, removed